cfgov/core/static/robots.txt

```diff
@@ -0,0 +1,6 @@
+User-agent: *
+Disallow: /save-hud-counselors-list
+Disallow: /about-us/the-bureau/leadership-calendar/pdf
+Disallow: /es/obtener-respuestas/*.imprimir.html
+
+sitemap: http://www.consumerfinance.gov/activity-log/feed/
```

Comment:
the sitemap protocol allows for RSS feeds to stand in: http://www.sitemaps.org/protocol.html#otherformats